module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderContentCard.kt

```diff
@@ -0,0 +1,47 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.ElevatedCard
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.github.ajalt.colormath.extensions.android.composecolor.toComposeColor
+import org.cru.godtools.shared.renderer.content.extensions.clickable
+import org.cru.godtools.shared.renderer.content.extensions.visibility
+import org.cru.godtools.shared.renderer.state.State
+import org.cru.godtools.shared.tool.parser.model.Card
+
+@Composable
+internal fun RenderContentCard(
+    card: Card,
+    state: State,
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(10.dp)
+) {
+    val scope = rememberCoroutineScope()
+
+    ElevatedCard(
+        colors = CardDefaults.cardColors(
+            containerColor = card.backgroundColor.toComposeColor(),
+        ),
+        elevation = CardDefaults.cardElevation(
+            defaultElevation = 6.dp
+        ),
```

Comment:
I actually don't think we need to exaggerate the card elevation this much for now, the default elevation for an `ElevatedCard` is `1dp`.

The `6dp` value is just shy of the default `8dp` for dragging a card.